docs/sources/tutorials/get-started.md

```diff
@@ -196,60 +201,58 @@ This last component creates a [loki.write][] component named `grafana_loki` that
 This completes the simple configuration pipeline.
 
 {{< admonition type="tip" >}}
-The `basic_auth` is commented out because the local `docker compose` stack doesn't require it. 
-It is included in this example to show how you can configure authorization for other environments.
+The `basic_auth` is commented out because the local `docker compose` stack doesn't require it.
```

Comment:
nit: "The `basic_auth` is commented out" reads weird to me; my thought goes to "the `basic_auth` _what_?" 

Should this spell out "The `basic_auth` block?"